script.js

```diff
@@ -1,4 +1,53 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+
+var scissors  = "scissors"
+var stone = "stone"
+var paper = "paper"
+
+//why must I use the work Main to mark out a function
+
+
+var randomnumber = function(){
+  var number = Math.floor(Math.random() * 2);
+
+  console.log("success one")
+  if (number == 0){
+    return scissors;
+  };
+
+  console.log("a")
+  if (number == 1){
+    return paper;
+    
+  };
+  console.log("b")
+  if (number == 2){
+    return stone
+  };
+}  
+
+var numbers = randomnumber()
```

Comment:
Wouldn't something like `rpsOption` be a better name for what you are trying to convey (instead of `numbers`)?
cause `randomNumber()` is returning either stone, scissors or paper right?